charts/redis-ha/values.yaml

```
@@ -430,6 +430,10 @@ additionalAffinities: {}
 ##
 affinity: |
 
+## https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/
+topologySpreadConstraints:
+  enabled: false
+
```

Can we have all the options visible in the values? 

```suggestion
topologySpreadConstraints:
  enabled: false
  maxSkew: ""
  topologyKey: ""
  whenUnsatisfiable: "" 
```

With this, think we're good to merge in.